draft-vvv-webtransport-quic.md

```diff
@@ -129,23 +131,70 @@ the client receives a TLS Finished message from the server.
 
 In order to identify itself as a WebTransport application, QuicTransport relies
 on TLS Application-Layer Protocol Negotiation {{!RFC7301}}.  The user agent MUST
-request the ALPN value of "wq" and it MUST NOT establish the session unless that
-value is accepted.
+request the ALPN value of "wq-draft01" and it MUST NOT establish the session
+unless that value is accepted.
+
+## Client Indication
+
+In order to verify that the client's origin is allowed to connect to the server
+in question, the user agent has to communicate the origin to the server.  This
+is accomplished by sending a special handshake message, called *client
+indication*, on stream 2, which is the first client-initiated unidirectional
+stream.
+
+The client indication is a sequence of key-value pairs that are formatted in the
+following way:
+
+~~~~~~~~~~ drawing
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                           Key (2)                           ...
```

Comment:
QUIC notation is in bits, not bytes. Also, can't we make these varints?

Reply:
Fixed the bytes one.  I've considered making those varints, but my concern was that not all QUIC libraries expose the varint parser to the applications, thus making this potentially harder to implement.

Reply:
Varints take about 10 lines of code to implement, I think they're worth the flexibility